plugins/georeplication/api/resp.go

```diff
@@ -0,0 +1,56 @@
+package georeplication
+
+import (
+	"github.com/pborman/uuid"
+)
+
+const (
+	// GeorepStatusCreated represents Created State
+	GeorepStatusCreated = "Created"
+
+	// GeorepStatusStarted represents Started State
+	GeorepStatusStarted = "Started"
+
+	// GeorepStatusStopped represents Stopped State
+	GeorepStatusStopped = "Stopped"
+
+	// GeorepStatusPaused represents Paused State
+	GeorepStatusPaused = "Paused"
+)
+
+// GeorepWorker represents Geo-replication Worker
+type GeorepWorker struct {
+	MasterNode                 string `json:"master_node"`
+	MasterNodeID               string `json:"node_id"`
+	MasterBrickPath            string `json:"master_brick_path"`
+	MasterBrick                string `json:"master_brick"`
+	Status                     string `json:"status"`
+	LastSyncedTime             string `json:"last_synced_time"`
+	LastSyncedTimeUTC          string `json:"last_synced_time_utc"`
+	LastEntrySyncedTime        string `json:"last_synced_entry"`
+	SlaveNode                  string `json:"slave_node"`
+	ChangeDetection            string `json:"change_detection"`
+	CheckpointTime             string `json:"checkpoint_time"`
+	CheckpointTimeUTC          string `json:"checkpoint_time_utc"`
+	CheckpointCompleted        bool   `json:"checkpoint_completed"`
+	CheckpointCompletedTime    string `json:"checkpoint_completed_time"`
+	CheckpointCompletedTimeUTC string `json:"checkpoint_completed_time_utc"`
+	MetaOps                    string `json:"meta"`
+	EntryOps                   string `json:"entry"`
+	DataOps                    string `json:"data"`
+	FailedOps                  string `json:"failures"`
+	CrawlStatus                string `json:"crawl_status"`
+}
+
+// GeorepSession represents Geo-replication session
+type GeorepSession struct {
+	MasterID   uuid.UUID         `json:"master_volume_id"`
```

Comment:
Check if the string representation of UUID is what that is seen in response.

Reply:
Yes. Sample output

HTTP/1.1 200 OK
Content-Type: application/json; charset=UTF-8
X-Request-Id: 52e4ac7a-a470-4a9c-952f-687bebe442f0
Date: Thu, 12 Oct 2017 07:26:23 GMT
Content-Length: 255

{"master_volume_id":"254218fe-37ac-40e7-9b76-980700217862","slave_volume_id":"bc8834b4-c710-4fd6-b07e-f25380595a62","master_volume":"gv1","slave_user":"root","slave_hosts":["f26"],"slave_volume":"gv3","monitor_status":"Created","workers":[],"options":{}}